app/sql/product_statuses.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO ProductStatus (id, name)
```

Comment:
Should be replaced with a boolean "listed" and moved into the Product table, unless we see a reason for having more than 2 types of statuses.